deployment/pom.xml

```diff
@@ -21,12 +21,12 @@
     <dependency>
       <groupId>io.quarkiverse.poi</groupId>
       <artifactId>quarkus-poi-deployment</artifactId>
-      <version>2.0.3</version>
+      <version>2.0.5</version>
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-jaxb-deployment</artifactId>
-      <version>3.3.2</version>
+      <version>${quarkus.version}</version>
```

Comment:
Nice to get rid of the hardcoded version. If I remeber correctly, we wanted to move to 3.3.2 for this dependency but quarkus 3.3.2 was not yet released